jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -21,55 +21,66 @@
 import java.nio.channels.ReadableByteChannel;
 import java.nio.file.Path;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Comparator;
 import java.util.HashSet;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import org.eclipse.jetty.util.IO;
-import org.eclipse.jetty.util.StringUtil;
 import org.eclipse.jetty.util.URIUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
- * A collection of resources (dirs).
- * Allows webapps to have multiple (static) sources.
+ * A collection of Resources.
+ * Allows webapps to have multiple sources.
  * The first resource in the collection is the main resource.
  * If a resource is not found in the main resource, it looks it up in
- * the order the resources were constructed.
+ * the order the provided in the constructors
  */
 public class ResourceCollection extends Resource
 {
-    private List<Resource> _resources;
+    private static final Logger LOG = LoggerFactory.getLogger(ResourceCollection.class);
 
-    /**
-     * Instantiates an empty resource collection.
-     * <p>
-     * This constructor is used when configuring jetty-maven-plugin.
-     */
-    public ResourceCollection()
+    public static class Mount implements Resource.Mount
```

Comment:
This should not be public.

Reply:
Set to package scope (so that `Resource` can use it)